src/hotspot/share/utilities/count_trailing_zeros.hpp

```diff
@@ -59,13 +65,23 @@ inline unsigned count_trailing_zeros(uintx x) {
 #pragma intrinsic(_BitScanForward)
```

Comment:
_BitScanForward needs to be "declared" intrinsic unconditionally, since it's now also used in _LP64 mode.